### PR TITLE
Add to onboarding reproduction logs

### PR DESCRIPTION
**Pull request by MojTabaa4**
### Adding reproduction logs for:
- start-here.md
- experiments-msmarco-passage.md

Environment:
Machine:  Google Colab (Ubuntu 22.04.2 LTS)
Processor:  x86_64
RAM: 12 GB

Python: 3.11.4
Python in Conda env: 3.8.18
Java: 11.0.20.1
Apache Maven: 3.6.3

Every result has been successfully reproducible and you can see them in this [notebook](https://github.com/MojTabaa4/castorini-onboarding-notebook).

### CODE REVIEW

File: docs/start-here.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ In standard parlance, a "document" is used generically to refer to any discrete
 We call them "documents" even though in reality they may be webpages, passages, PDFs, Powerpoint slides, Excel spreadsheets, or even images, audio, or video.
 
 The output of retrieval is a ranking of documents (i.e., a ranked list, or just a sorted list).
-Documents are identified by unique ids, and so a ranking is simply a list of ids.
+Documents are identified by unique IDs, and so a ranking is simply a list of IDs.
```
Comment (lintool) [inline docs/start-here.md:78]: We've been using "id" consistently lowercase, so can you please change back?

File: docs/start-here.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ q2 doc97 0
 
 That is, for `q1`, `doc23` is not relevant, `doc452` is relevant, and `doc536` is not relevant; for `q2`, `doc97` is not relevant...
 
-Now, given a set of queries, you feed each query into your retrieval system and get back a ranked list of document ids.
+Now, given a set of queries, you feed each query into your retrieval system and get back a ranked list of document IDs.
```
Comment (lintool) [inline docs/start-here.md:107]: same

File: docs/start-here.md [hunk 3]
```diff
@@ -249,7 +249,7 @@ These are the queries in the development set of the MS MARCO passage ranking tes
 The first field is a unique identifier for the query (called the `qid`) and the second column is the query itself.
 These queries are taken from Bing search logs, so they're "realistic" web queries in that they may be ambiguous, contain typos, etc.
 
-Okay, let's now cross reference the `qid` with the relvance judgments, i.e., the qrels file:
+Okay, let's now cross-reference the `qid` with the relevance judgments, i.e., the qrels file:
```
Comment (lintool) [inline docs/start-here.md:252]: good catch!

**Review by lintool — CHANGES_REQUESTED** [review]

File: docs/start-here.md (hunk 3 above)
Comment (MojTabaa4) [reply docs/start-here.md:252]: Thanks, I reverted IDs to ids, apologize for the mistake.

Comment (lintool) [reply docs/start-here.md:252]: no worries! no way you would have known...

**Comment by lintool** [comment]
> I'd like to know your opinion about my notebook. Is it OK to make [this repo](https://github.com/MojTabaa4/castorini-onboarding-notebook) public?

no strong feelings one way or another... kinda nice to have everything in one place... but, also, making all other repro efforts too easy if they can just just copy/paste or clone repo...

**Review by lintool — APPROVED** [review]

**Pull request merged**